### PR TITLE
Fix sourcemap comments to be used on file transformers with CSS output

### PR DESCRIPTION
**Pull request by IvanSanchez**
This fixes a case where a file transformer (namely, `gobble-sass-all`) takes `.sass` files as input and `.css` files as output - the code was (wrongly) assuming that the input file extension decides which sourcemap comment style to use. In other words: my CSS files were being written a javascript-style sourcemap comment.

### CODE REVIEW

**Comment by IvanSanchez** [comment]
@Rich-Harris I'll leave the `npm publish` magic up to you.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "gobble",
   "description": "The last build tool you'll ever need",
-  "version": "0.11.3",
+  "version": "0.11.4",
```
Comment (TrySound) [inline package.json]: Can you revert this change and squash commits?

Comment (IvanSanchez) [inline package.json]: I'll try rebasing.

**Comment by IvanSanchez** [comment]
@TrySound This has been rebased against `master`.

**Comment by TrySound** [comment]
Cool, thanks!
